crates/oxc_language_server/src/main.rs

```diff
@@ -1,6 +1,6 @@
-#[allow(clippy::disallowed_types)]
 mod linter;
 
+#[allow(clippy::disallowed_types)]
```

Comment:
```suggestion
#[expect(clippy::disallowed_types)]
```

A very minor point: `#[expect]` is preferable to `#[allow]`, as if we change linting rules in future and it makes this attribute defunct, then linter will flag that it can be removed.

Reply:
Maybe we should disallow `#[allow]`!